twitter.py

```diff
@@ -21,13 +21,15 @@
 
 f = open(filepath, "a+")
 regex = r"(^|[^@\w])@(\w{1,30})"
+newtweetcounter, oldtweetcounter = 0, 0
 
 for tweet in tweepy.Cursor(api.search,q="concours RT",
                            lang="fr",result_type='popular', tweet_mode='extended').items(int(os.getenv('SEARCH_ITEM_NUMBER'))):
 
     if int(tweet.id_str) in listTweetID:
         if os.getenv('DEBUG') == 1:
```

Comment:
looks like the getenv return a string so the condition is always false here

Reply:
This issue is fixed in the last commit, thanks for reporting it